src/main/java/ja/workshops/spring/części/konfiguracja/xml/c_mvc/SerwisSamochodowy.java

```diff
@@ -0,0 +1,19 @@
+package ja.workshops.spring.części.konfiguracja.xml.c_mvc;
+
+/**
+ * Serwis, który jest warstwą łączącą dane wyciągane z repozytorium {@link RepozytoriumSamochodów} oraz warstwą prezentacji {@link KomisKontroler}.
```

Comment:
W rozumieniu warstw z wielowarstwowej archi to można by się z Tobą spierać czy serwis jest warstwą. Łącznik, komponent (ale tu znowu ryzyko bo Springowy?) albo usługa (choć to po prostu powtarza serwis). Moja propozycja:

Wyciąga dane z repozytorium i udostępnia je warstwie prezentacji.